kr/debug/gdb_debugging.md

```diff
@@ -128,7 +126,7 @@ EXC_RETURN: ffffffe9
 <div class="host-code"></div>
 
 ```bash
-arm-none-eabi-gdb build/px4fmu-v2_default/src/firmware/nuttx/firmware_nuttx
+arm-none-eabi-gdb build/px4_fmu-v2_default/src/firmware/nuttx/firmware_nuttx
```

Comment:
Is the rest of the path in this still oK? ... /src/firmware/nuttx/firmware_nuttx

Reply:
./build/px4_fmu-v2_default/px4_fmu-v2_default.elf

Reply:
Thanks. Fixed in https://github.com/PX4/Devguide/commit/d0443c97fe204da65b9d4b34e54788f4aabaa8d0